oshi-core/src/main/java/oshi/software/os/mac/MacOperatingSystem.java

```diff
@@ -154,6 +157,18 @@ private String parseCodeName() {
                 return "Cheetah";
             default:
             }
+            */
+            final String OSHI_VERSIONS_PROPERTIES = "oshi.versions.properties";
```

Comment:
Declare this `public static` at the top of the file.  Also probably want `macos.versions`.